frame/instrumental-strategy-pablo/src/lib.rs

```diff
@@ -198,7 +216,26 @@ pub mod pallet {
 	// ---------------------------------------------------------------------------------------------
 
 	#[pallet::call]
-	impl<T: Config> Pallet<T> {}
+	impl<T: Config> Pallet<T> {
+		/// Store a mapping of asset_id -> pool_id in the pools runtime storage object.
+		///
+		/// Emits [`Setted`](Event::Setted) event when successful.
```

Comment:
I have not found an implementation of this. Could you point me out please?

Reply:
https://github.com/ComposableFi/composable/pull/1187/files#:~:text=Setted%20%7B%20asset_id%3A%20T%3A%3AAssetId%2C%20pool_id%3A%20T%3A%3APoolId%20%7D%2C

Reply:
Link don't work for me. We already discuss it in Slack: Firefox does not support this type of links. Could you create permanent link using GitHub please?

Reply:
Ofk! [Link](https://github.com/ComposableFi/composable/blob/a30dc18fe2a34d1b5c83f149c9a2db0bbc886435/frame/instrumental-strategy-pablo/src/lib.rs#L185)

Reply:
Oh, sorry. I mean, I have not found of an usage of this event. I have not found an implementation of something when event will be successful (as stated in the documentation).

Reply:
Oh, it's my bad. I forgot to add it. Thanks for your attention! 
I [added it](https://github.com/ComposableFi/composable/blob/d7fb07f3d854c8ffa3af14e27dea74013c0ce583/frame/instrumental-strategy-pablo/src/lib.rs#L237) now.